src/gui/widgets/rich_label.hpp

```diff
@@ -216,21 +223,21 @@ class rich_label : public styled_widget
 	}
 
 	/** size calculation functions */
-	point get_text_size(config text_cfg, unsigned width = 0);
-	point get_image_size(config img_cfg);
+	const point get_text_size(const config& text_cfg, const unsigned width = 0);
+	const point get_image_size(const config& img_cfg);
```

Comment:
It's generally not good to have methods returning const values. Suggest just keeping these returning `point`

Reply:
In addition, that const in the parameter list (`const unsigned`) is literally meaningless – the compiler will ignore it. The toplevel const-ness of a function parameter is not part of the function's signature. (It does have meaning in the function _definition_, mind you, though I don't think it's a very useful thing to do.)

Reply:
reverted to previous version.